jsonnet/utils/add-labels.libsonnet

```
@@ -0,0 +1,19 @@
+{
+  local managedBy(o) =
+    local name = o.metadata.name;
+    // Some of the resources which are generated by jsonnet are moved under manifests/ and managed by CVO.
+    if o.kind == 'CustomResourceDefinition' || (o.kind == 'Role' && name == 'cluster-monitoring-operator-alert-customization') || (o.kind == 'Deployment' && name == 'cluster-monitoring-operator') || (o.kind == 'ClusterRole' && std.setMember(name, ['cluster-monitoring-operator-namespaced', 'cluster-monitoring-operator'])) then
+      'cluster-version-operator'
+    else
+      'cluster-monitoring-operator',
+
+  addLabels(o, labels): {
```

```suggestion
  addLabels(labels,o ): {
```
Nit but what do you think of swapping the arguments here. I think that would improve readability in the main file. Other wise we have
```
addLabels(
    ...32 lines...,
    $labelsToAdd)

---

done